### ISSUE TITLE
Shared head cotrain: Aria (12D actions) fails to converge while EVA (14D actions) converges well

### ISSUE THREAD

**Issue by karry-yu** [GaTech-RL2/EgoVerse#533]
### Topic

Training Configuration

### Question

We're cotraining HPT + Flow Matching on two bimanual embodiments (Aria 12D+padding, EVA 14D) with a shared CrossTransformer head. EVA converges well but Aria metrics barely move after epoch 200. Aria-only training converges fine.
Epoch | EVA Fréchet | Aria Fréchet |  EVA KL | Aria KL | EVA MSE | Aria MSE
 199  |      0.554  |       1.059  |   67.4  |  164.5  |  0.045  |  0.140  
 1799 |      0.100  |       1.053  |   10.9  |  143.5  |  0.007  |  0.127  

Any suggestions?

**Comment by AnikethCheluva** [comment]
What was the data mixture you were using?

**Comment by karry-yu** [comment]
@AnikethCheluva 
`eva-pick-place` and `aria-bimanual`(only pick place task) 
here is the config:
```yaml
train_datasets:
  eva_bimanual:
    _target_: egomimic.rldb.zarr.zarr_dataset_multi.MultiDataset._from_resolver
    resolver:
      _target_: egomimic.rldb.zarr.zarr_dataset_multi.LocalEpisodeResolver
      folder_path: ${paths.dataset_dir}/eva-pick-place
      key_map:
        _target_: egomimic.rldb.embodiment.eva.Eva.get_keymap
        keymap_mode: cartesian
      transform_list:
        _target_: egomimic.rldb.embodiment.eva.Eva.get_transform_list
        mode: cartesian
    filters: null
    mode: total
  aria_bimanual:
    _target_: egomimic.rldb.zarr.zarr_dataset_multi.MultiDataset._from_resolver
    resolver:
      _target_: egomimic.rldb.zarr.zarr_dataset_multi.LocalEpisodeResolver
      folder_path: ${paths.dataset_dir}/aria-bimanual
      key_map:
        _target_: egomimic.rldb.embodiment.human.Aria.get_keymap
        keymap_mode: cartesian
      transform_list:
        _target_: egomimic.rldb.embodiment.human.Aria.get_transform_list
        mode: cartesian
    filters:
      _target_: egomimic.rldb.filters.DatasetFilter
      filter_lambdas:
        - "lambda row: row.get('task_name', '') == 'pick_place'"
        - "lambda row: row.get('episode_hash', '') != '2026-03-17-17-30-47-000000'"
    mode: total

```
